### PR TITLE
Ensure service is fastboot compatible. Specifically with using `scrol…

### PR DESCRIPTION
**Pull request by snewcomer**
…lElement`

### CODE REVIEW

File: addon/services/router-scroll.js [hunk 1]
```diff
@@ -26,6 +30,10 @@ export default Service.extend({
       x = window.scrollX;
       y = window.scrollY;
     } else if ('#' === scrollElement.charAt(0)) {
+      if (get(this, 'isFastBoot')) {
```
Comment (snewcomer) [inline addon/services/router-scroll.js]: Not sure if this should go at the top of the `update` method....

**Comment by rwwagner90** [comment]
@snewcomer shouldn't we also be checking for fastboot in this file https://github.com/dollarshaveclub/ember-router-scroll/blob/master/addon/index.js? There are a few document and window usages in there. FWIW, I have never encountered a fastboot error using this addon, but I guess most of my usages were after the browser took over.

**Comment by snewcomer** [comment]
@rwwagner90 I think that is covered [here](https://github.com/dollarshaveclub/ember-router-scroll/blob/master/addon/index.js#L26), so prevents that code from ever running.

I wanted to use `scrollElement` (figured out I need #99), but still got an error on fastboot with `scrollElement`.  

I think I should add some tests around that service as well for `scrollElement`

**Comment by rwwagner90** [comment]
Ah, I see. Tests would be great!

**Comment by rwwagner90** [comment]
Thanks!

**Pull request merged**